GETTING_STARTED.md

```diff
@@ -186,6 +186,31 @@ Another advantage of this approach is that if you need to use a `DbContext` insi
 * `WebActivatorEx`'s `PreApplicationStartMethod` (the `PreStart` method in our sample above) runs **before** OWIN's Startup method.
 	* See this StackOverflow post: https://stackoverflow.com/questions/21462777/webactivatorex-vs-owinstartup
 
+### How to deal with `ObjectDataSource` controls
+Background: `ObjectDataSource` controls invoke a `Select` method defined in the `Page` object in order to retrieve the data to bind to other controls. This select method is usually a `static` method on the containing page, which prevents using DI, but can also be an instance method, in which case it instantiates a new `Page` object and calls the method on the newly created instance. That's the scenario when DI should be possible.
+
+Unfortunately it seems that Microsoft never updated the `ObjectDataSource` control to use `HttpRuntime.WebObjectActivator`, so it's not able to instantiate DI-enabled `Page` controls. However, it is possible to override its `ObjectCreating` event to make it work the way we want:
+
+```
+using System.Web;
+using System.Web.Compilation;
+using System.Web.UI.WebControls;
+...
+public class DiObjectDataSource: ObjectDataSource 
+{
+  public DiObjectDataSource()
+  {
+#if NET472_OR_GREATER
```

Comment:
The .NET Fx 4.7.2 check is redundant as the entire `AspNetDependencyInjection` project has that as a minimum version requirement already.

Reply:
My mistake, I left some older code in there. Thanks for spotting!